cloud/azure/variables.tf

```diff
@@ -20,6 +20,18 @@ variable "poll_rate" {
   default     = 300
 }
 
+variable "host_or_usage_limits" {
+  description = "Specify Usage-based limits for this integration"
+  type        = map(number)
```

Comment:
Why not an object since all map keys are known ?

Reply:
because an object will force user to define ALL fields which is exactly what we do not want